### ISSUE TITLE
Unsupported keras layer type Bidirectional

### ISSUE THREAD

**Issue by thomasjungblut** [deeplearning4j/deeplearning4j#3519]
Hey,

I just tried to import a simple bi-lstm sequential model:

```python
def nn_model():
    m = Sequential()

    m.add(Bidirectional(LSTM(5, return_sequences=True), input_shape=(5, 10)))
    m.add(TimeDistributed(Dense(5, activation='softmax')))
    m.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['categorical_accuracy'])
    return m
```  

And the resulting hdf5 model back in Java using:

```java
this.network = KerasModelImport.importKerasSequentialModelAndWeights(hdf5FilePath);
``` 

and I get the exception:
``` 
Caused by: org.deeplearning4j.nn.modelimport.keras.UnsupportedKerasConfigurationException: Unsupported keras layer type Bidirectional. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getKerasLayerFromConfig(KerasLayer.java:254)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperPrepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:99)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:56)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel$ModelBuilder.buildSequential(KerasModel.java:581)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:156)
``` 

There is a bi-lstm in dl4j, but it doesn't seem to be mapped to it. Any plans on adding support for it? Will there be issues with the ```TimeDistributed``` layer, too?

#### Version Information
deeplearning4j-modelimport:0.8.0-2
Keras (2.0.4)
h5py (2.6.0)
tensorflow (1.1.0)

**Comment by AlexDBlack** [comment]
DL4J bidirecitonal wrapper layer: https://github.com/deeplearning4j/deeplearning4j/pull/4396

**Comment by maxpumperla** [comment]
@thomasjungblut we've just merged #4410, you're good to go on master now.
